### ISSUE TITLE
How to display a proper bullet list?

### ISSUE THREAD

**Issue by iupchris10** [SuavePirate/MarkdownTextView#6]
I am trying to figure out how to display a simple bullet list with MarkdownTextView. I've tried several combinations of strings and can't get any of them to display the list correctly.  Got any tips to make this work?

![screen shot 2018-03-07 at 10 27 18 am](https://user-images.githubusercontent.com/5750525/37100925-46be28b6-21f2-11e8-806a-eb6c2e3b07f6.png)

**Comment by SuavePirate** [comment]
You need to add in-string line breaks with \n since the line breaks in your code won't reflect that in the actual string.

**Comment by iupchris10** [comment]
Yeah, I've tried that also.  Here are two other attempts. It seems to double-bullet the list when I do this.

![screen shot 2018-03-07 at 11 54 54 am](https://user-images.githubusercontent.com/5750525/37105972-8d20f35e-21fe-11e8-9a30-db1301692f0f.png)
![screen shot 2018-03-07 at 11 55 22 am](https://user-images.githubusercontent.com/5750525/37105973-8d2df270-21fe-11e8-8519-9995529f9920.png)

**Comment by SuavePirate** [comment]
Interesting. Is this just on Android? What versions of Android are you seeing this? FYI, this class is the culprit for causing issues with the list and is up for some major refactoring since it is a port from a Java class with a few adjustments: https://github.com/SuavePirate/MarkdownTextView/blob/master/src/Forms/SPControls.MarkdownTextView/SPControls.MarkdownTextView.Droid/HtmlTagHandler.cs

The <ul> and <ol> handling on Android is not built into the FormattedText set up, so it takes a bunch of custom logic to work it in, so if I can't get to fixing this in time, I would suggest breaking that one out to find your issue.

It's also worth noting that I'm rebuilding this control with a whole new approach now that it has started picking up popularity to make it more customizable, faster, and up to what I expect from it.
